virt/virtual_machines/advanced_vm_management/virt-configuring-default-cpu-model.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="virt-configuring-default-cpu-model"]
+= Configuring the default CPU model
+include::_attributes/common-attributes.adoc[]
+:context: virt-configuring-default-cpu-model
+
+Use the `defaultCPUModel` setting in the `HyperConverged` custom resource (CR) to define a cluster-wide default CPU model.
+
+The virtual machine (VM) CPU model depends on the availability of CPU models within the VM and the cluster.
+
+* If the VM does not have a defined CPU model:
+** The `defaultCPUModel` is automatically set using the CPU model defined at the cluster-wide level.
+* If both the VM and the cluster have a defined CPU model:
+** The VM’s CPU model takes precedence.
+* If neither the VM nor the cluster have a defined CPU model:
+** The host-model is automatically set using the CPU model defined at the host level.
```

Comment:
A shorter alternative to explain that could be something like:
```
If a CPU model is defined at the VM level, it would take precedence.
Otherwise, the model would be determined by whatever's defined at the cluster-wide level.
If neither are defined, the default CPU model is `host-model`.
```

Whatever you think is better

Reply:
Going to leave this one as-is, because I think the wording and bulleted list make it a little more clear, but I'm open if the docs peer reviewer has any thoughts here.